### DISCUSSION TITLE
CAN FD log and playback

### DISCUSSION THREAD

**Discussion by Camelrider79** [collin80/SavvyCAN#742] [Q&A]
Hello everyone, I am currently trying to send ID's logged by a Kia EV6 over can fd, files via playback function to the kia's battery. Is it possible that the Savvycan only does the playback via can, instead of can-fd?

**Comment by kukumagi** [comment]
Hi. Not really realted to this post. But have you already got a connection to EV6 battery can?

**Reply by Camelrider79** [reply]
Yes, I have, do you need anything?

**Reply by kukumagi** [reply]
I already noticed your posts in Battery-emulator project I am going to continue from there.
And meanwhile I also got successful data with mcp2515 + arduino + canhacker.

**Reply by Camelrider79** [reply]
come in to Dala's discord, you can have also all data from the car. We can better discuss the next steps.

**Comment by mg-nz** [comment]
I was just about to try something very similar - can you share the EV6 CAN FD log? I am trying to get communications working with some other parts from the Powertrain CAN bus so I can test it and see whether I get FD playback

**Reply by kukumagi** [reply]
Part of  a log with some sort of timestamps (Not mine):
108980RX48100284B0x000001cf80xe7 60 00 20 00 00 00 00 Default Parser
1015722RX48100286B0x0000019a320x6d ed 69 55 44 64 d8 1b 40 20 00 00 00 00 11 52 00 12 02 64 00 00 00 08 13 00 00 00 00 32 00 00 Default Parser
1017959RX48100289B0x00000035320x20 28 7b 01 00 00 20 01 00 00 00 10 00 40 00 00 06 8d 00 00 f0 39 01 00 09 00 00 00 00 00 00 00 Default Parser
1017958RX48100289B0x000000da240x39 89 7b 00 00 00 00 00 01 00 00 00 00 00 00 00 00 00 00 00 00 00 5d 02 Default Parser
